### DISCUSSION TITLE
安装了esbuild失败卸载后mfsu不能使用

### DISCUSSION THREAD

**Discussion by mschenhe** [umijs/umi#7475] [Q&A]
安装了esbuild失败卸载后mfsu不能使用，启动报错，把mfsu:{}注释掉之后又可以启动了，感觉有点玄学，想请问下大佬们，这是怎么回事呢？
![image](https://user-images.githubusercontent.com/31650065/137681849-8e70c63b-63a9-425c-b926-cc154867cb8b.png)
![image](https://user-images.githubusercontent.com/31650065/137681887-8b0d5e6d-955f-4578-8ae8-91c50bd8dfef.png)
![image](https://user-images.githubusercontent.com/31650065/137681997-f57d74a3-45ad-45a0-abf6-deb8aa600da7.png)

**Comment by renkunx** [comment]
清除一下.umi 等缓存文件试试
